step.yml

```diff
@@ -32,48 +20,77 @@ run_if: ""
 
 deps:
   brew:
-  - name: getsentry/tools/sentry-cli
-
+    - name: getsentry/tools/sentry-cli
 
 toolkit:
-  bash:
-    entry_file: step.sh
-
+  go:
+    package_name: github.com/SimonRice/bitrise-step-sentry-dsym-upload
 
 inputs:
-  - sentry_auth_token: $SENTRY_AUTH_TOKEN
+  - platform: both
+    opts:
+      title: "Platform"
+      summary: "The selected platform that symbols will be uploaded for, or both iOS and Android"
+      description: "The selected platform that symbols will be uploaded for, or both iOS and Android"
+      is_required: true
+      value_options:
+        - "both"
+        - "ios"
+        - "android"
+  - is_debug_mode: "false"
+    opts:
+      title: "Debug mode?"
+      summary: "If debug mode is enabled, the step will print verbose logs"
+      description: |-
+        If debug mode is enabled, the step will print verbose logs
+      is_required: true
+      value_options:
+        - "true"
+        - "false"
+  - auth_token:
     opts:
-      title: Auth token for Sentry
+      title: Auth token for your Sentry user account
+      summary: Auth token for your Sentry user account. Required to upload symbols.
       description: |
-        Auth token for Sentry
+        "Auth token can be created on Sentry via Settings > Account > API > Auth Tokens"
       is_required: true
       is_expand: true
       is_sensitive: true
+
   - sentry_url: https://sentry.io/
     opts:
       title: Server URL for Sentry
-      description: |
+      summary: |
         Fully qualified URL to the Sentry server.
         [defaults to https://sentry.io/]
       is_required: true
-  - sentry_org_slug:
+
+  - org_slug:
     opts:
-      title: Org Slug for Sentry
-      description: |
-        Org Slug
+      title: Organisation slug
+      summary: "Organisation slug for your Sentry organisation"
       is_required: true
       is_expand: true
-  - sentry_project_slug:
+      is_sensitive: true
+
+  - project_slug:
     opts:
-      title: Project Slug for Sentry
-      description: |
-        Project Slug
+      title: Project slug
+      summary: "Project slug for your Sentry project"
       is_required: true
       is_expand: true
-  - sentry_dsym_path: $BITRISE_DSYM_PATH
+      is_sensitive: true
+
+  - dsym_path:
     opts:
-      title: Path to your dSYM file
-      description: |
-        Path to your dSYM file
-      is_required: true
+      title: dSYM path
+      summary: "Path to your dSYM"
       is_expand: true
+      is_sensitive: true
+
+  - proguard_mapping_path:
+    opts:
+      title: Proguard mapping.txt path
+      summary: "Path to your Proguard mapping.txt"
+      is_expand: true
+      is_sensitive: true
```

Comment:
I'm curious about whether dSYM & Proguard paths need to be sensitive here - I can understand why you made project & organisation slugs sensitive.

Reply:
yeah probably not necessary, I'm guilty of a copypasta here that I forgot to change

Reply:
I tend to call it "copy & haste" 🙃